src/pkg/cluster/injector.go

```diff
@@ -56,43 +59,38 @@ func (c *Cluster) StartInjectionMadness(ctx context.Context, tmpDir string, imag
 	}
 
 	if err := helpers.CreateDirectory(tmp.SeedImagesDir, helpers.ReadWriteExecuteUser); err != nil {
-		spinner.Fatalf(err, "Unable to create the seed images directory")
+		return fmt.Errorf("unable to create the seed images directory: %w", err)
 	}
 
 	var err error
 	var images imageNodeMap
 	var payloadConfigmaps []string
 	var sha256sum string
-	var seedImages []transform.Image
 
-	// Get all the images from the cluster
-	spinner.Updatef("Getting the list of existing cluster images")
```

Comment:
Why did we get rid of all the spinner updates?

Reply:
Because they are spammy and low value to the user IMO

Reply:
Also I didn't remove all of them. For example I kept the ones that update the user when long-running operations are happening, like creating the payload configmaps

Reply:
Yeah good point, I like this better